vmm/sandbox/src/qemu/mod.rs

```diff
@@ -70,21 +71,15 @@ pub(crate) const QEMU_START_TIMEOUT_IN_SEC: u64 = 10;
 // but skip all the fields serialization.
 #[derive(Default, Serialize, Deserialize)]
 pub struct QemuVM {
-    #[serde(skip)]
     id: String,
-    #[serde(skip)]
     config: QemuConfig,
     #[serde(skip)]
     devices: Vec<Box<dyn QemuDevice + Sync + Send>>,
     #[serde(skip)]
     hot_attached_devices: Vec<Box<dyn QemuHotAttachable + Sync + Send>>,
-    #[serde(skip)]
     fds: Vec<RawFd>,
-    #[serde(skip)]
     console_socket: String,
-    #[serde(skip)]
     agent_socket: String,
-    #[serde(skip)]
     netns: String,
     #[serde(skip)]
     block_driver: BlockDriver,
```

Comment:
maybe we also have to recover this field so that we can decide which block driver to use for block device hot attach?

Reply:
fixed